.github/workflows/mirror-to-gitlab.yml

```
@@ -0,0 +1,33 @@
+          GITLAB_URL: ${{ secrets.GITLAB_URL }}
+          GITLAB_TOKEN: ${{ secrets.GITLAB_TOKEN }}
+          GITLAB_USERNAME: ${{ secrets.GITLAB_USERNAME }}
+          GITLAB_REPO: ${{ secrets.GITLAB_REPO }}
+          BRANCH: ${{ github.ref_name }}
+        run: |
+          git config --global user.name "${GITLAB_USERNAME}"
+          git config --global user.email "${GITLAB_USERNAME}@users.noreply.gitlab.com"
+
+          git remote add gitlab https://${GITLAB_USERNAME}:${GITLAB_TOKEN}@${GITLAB_URL#https://}/${GITLAB_REPO}.git
+
+          git push -u --force gitlab HEAD:${BRANCH}
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)